### ISSUE TITLE
Uninstrumented modules in New Relic agent initialization

### ISSUE THREAD

**Issue by jgmize** [mozilla/bedrock#7372]
## Description

Getting the following error on https://rpm.newrelic.com/accounts/1299394/applications/57236602 since the latest prod deploy:

> Some modules were uninstrumented during the current time window: meinheld.server, gunicorn.app.base. Make sure newrelic.agent.initialize() is called before any of these modules are imported. Visit the [documentation](https://docs.newrelic.com/docs/agents/python-agent/installation-configuration/python-agent-integration#manual-integration) for more information.

This appears to be a regression since the Django-2.2/Python3 upgrade.

**Comment by ipmb** [comment]
I think this is unrelated to the upgrade and was probably always the case with how newrelic is currently being invoked, but just now being reported.

The only way around it I know of is to use the wrapper `newrelic-admin run-program` to start the gunicorn process. Right now, gunicorn starts, then imports the WSGI app, which then invokes `newrelic.initialize`, so we expect it can't instrument gunicorn which was executed before newrelic was initialized.

All that being said, it's also probably safe to ignore this warning. Newrelic is instrumenting your Django code, and it not being able to instrument those modules probably won't have any significant effect on the reports you see.

**Comment by jgmize** [comment]
Thanks @ipmb. I agree with your reasoning that it is probably safe to ignore this warning, and the `newrelic-admin` is likely the only way to remove it. In fact, I shared very similar thoughts last week in a discussion with @SmileyChris  & @metadave. 

For background, we [ran into similar issues before](https://github.com/mozilla/bedrock/pull/7149), and after a couple of [iterations](https://github.com/mozilla/bedrock/pull/7150) that issue had not reappeared until the Django 2.2/Python3 upgrade was deployed. When I first saw this appear during the initial deployment of the upgrade to our development & staging environments, I decided it was not a blocker for going to production for the same reasons you listed above, and I consider this to be a low priority issue that is not at all blocker for considering the upgrade work complete.
